images/rootfs.yml.in

```diff
@@ -26,6 +26,10 @@ onboot:
    - name: storage-init
      image: STORAGE_INIT_TAG
 services:
+   - name: edgeview
```

Comment:
Hey @naiming-zededa -- I don't think including edge-view in the default build is a good idea. Let's discuss separately how can we have an architecture where this container can only be deployed via controller.

Reply:
@rvs  The container although it's running, but it does not do anything, unless the controller sends the command to assign token, ip:port, timeout, then the script inside the container will launch the real 'edge-view' program to run. And the script will kill it if the timeout expires. Will this be ok?